<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>796</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFILE">
    <property name="title">
     <string>FILE</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionRecent_files"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEDIT">
    <property name="title">
     <string>EDIT</string>
    </property>
    <addaction name="actionGrayScale_Luminance"/>
    <addaction name="actionGrayScale_Average"/>
    <addaction name="actionSepia"/>
    <addaction name="actionBlur"/>
    <addaction name="actionNegative"/>
    <addaction name="actionTwo_Tones"/>
    <addaction name="actionEmboss"/>
    <addaction name="actionBrightness"/>
    <addaction name="actionTV_60"/>
   </widget>
   <widget class="QMenu" name="menuHELP">
    <property name="title">
     <string>HELP</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionInfo"/>
   </widget>
   <addaction name="menuFILE"/>
   <addaction name="menuEDIT"/>
   <addaction name="menuHELP"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_as"/>
   <addaction name="actionRecent_files"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normaloff>img/open.png</normaloff>img/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normaloff>img/save.png</normaloff>img/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset>
     <normaloff>img/save as.jpg</normaloff>img/save as.jpg</iconset>
   </property>
   <property name="text">
    <string>Save as</string>
   </property>
   <property name="shortcut">
    <string>Shift+S</string>
   </property>
  </action>
  <action name="actionRecent_files">
   <property name="icon">
    <iconset>
     <normaloff>img/recent file.png</normaloff>img/recent file.png</iconset>
   </property>
   <property name="text">
    <string>Recent files</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <normaloff>img/exit.jpg</normaloff>img/exit.jpg</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset>
     <normaloff>img/about.jpg</normaloff>img/about.jpg</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>Shift+A</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="icon">
    <iconset>
     <normaloff>img/info.png</normaloff>img/info.png</iconset>
   </property>
   <property name="text">
    <string>Info</string>
   </property>
   <property name="shortcut">
    <string>Shift+I</string>
   </property>
  </action>
  <action name="actionGrayScale_Luminance">
   <property name="text">
    <string>GrayScale Luminance</string>
   </property>
  </action>
  <action name="actionGrayScale_Average">
   <property name="text">
    <string>GrayScale Average</string>
   </property>
  </action>
  <action name="actionSepia">
   <property name="text">
    <string>Sepia</string>
   </property>
  </action>
  <action name="actionBlur">
   <property name="text">
    <string>Blur</string>
   </property>
  </action>
  <action name="actionNegative">
   <property name="text">
    <string>Negative</string>
   </property>
  </action>
  <action name="actionTwo_Tones">
   <property name="text">
    <string>Two Tones</string>
   </property>
  </action>
  <action name="actionEmboss">
   <property name="text">
    <string>Emboss</string>
   </property>
  </action>
  <action name="actionBrightness">
   <property name="text">
    <string>Brightness</string>
   </property>
  </action>
  <action name="actionTV_60">
   <property name="text">
    <string>TV_60</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
